src/test/bash/test-docker-build-on-release-branch.sh

```diff
@@ -8,5 +8,6 @@ set -ev
 
 if [[ $TRAVIS_BRANCH == release-* ]]
 then
+  git checkout settings.gradle
```

Comment:
Perhaps better to remove this and move the line that modifies `settings.gradle` in the `.travis.yml` file to the end of the `script` section, because during deployment we will still want to publish the `managementportal-client` library.

Reply:
Makes sense